addons/website_popup/static/src/js/website_popup.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+    'use strict';
+    var website = openerp.website,
+        qweb = openerp.qweb;
+
+    if (!website.snippet) website.snippet = {};
+    website.snippet.animationRegistry.banner = website.snippet.Animation.extend({
+        selector: ".banner_popup",
+        start: function (editable_mode) {
+            var self = this;
+            self.redirect_url = false;
+            var popupcontent = self.$target.find(".popup_content").empty();
+
+            openerp.jsonRpc('/website_popup/get_content', 'call', {
+                list_id: self.$target.data('list-id'),
+            }).then(function (data) {
+                $(data.content).appendTo(popupcontent)
+                self.redirect_url = data.redirect_url
+            });
+
+            if (!editable_mode) {
+                if (!self.$target[0].isContentEditable) {
+                    $(document).on('mouseleave', _.bind(self.show_banner, self));
+                } else { $(document).off('mouseleave'); }
+
+                self.$target.find('.popup_subscribe_btn').on('click', function (event) {
+                    event.preventDefault();
+                    self.on_click_subscribe();
+                });
+            }

```

Comment:
Hi @mba-odoo, in website_snippet_editor.js you can pass editable_mode paramater to avoid *if (!self.$target[0].isContentEditable) {* this type of condition and when website page is more with content  than popup needs to scroll to see.